tests/test_docs.py

```diff
@@ -57,23 +57,23 @@ def test_device_docstrings(self):
         for dev in self.devices:
```

Comment:
whoa, didn't realize christian had made tests for docs!

Reply:
I was shocked too!